browser/policy/brave_simple_policy_map.h

```diff
@@ -59,6 +60,10 @@ inline constexpr PolicyToPreferenceMapEntry kBraveSimplePolicyMap[] = {
 #endif
     {policy::key::kBraveAIChatEnabled, ai_chat::prefs::kEnabledByPolicy,
      base::Value::Type::BOOLEAN},
+    {policy::key::kBraveNewsDisabled,
```

Comment:
whoops, I think this is supposed to be inside the guard

Reply:
Fixed and rebased in cdbaf213770eeb46d2c4e8eff457da0ec932060d